.docker/.env

```diff
@@ -6,7 +6,6 @@ PROFILE=default
 # Local storage service environment variables
 LOCAL_STORAGE_SERVICE_DATABASE_USER=admin
 LOCAL_STORAGE_SERVICE_DATABASE_PASSWORD=admin
-LOCAL_STORAGE_SERVICE_DISK_MIN_SPACE_THRESHOLD=5GB
 
 RABBITMQ_DEFAULT_USER=admin
```

Comment:
Artem recomended change name of .env variable RABBITMQ_DEFAULT_USER to RABBITMQ_USER and RABBITMQ_DEFAULT_PASSWORD to RABBITMQ_PASSWORD, because we can have mistakes in dockerc-compose file when we use the same names like RABBITMQ_DEFAULT_PASSWORD: ${RABBITMQ_DEFAULT_PASSWORD}